src/js/components/action.jsx

```diff
@@ -112,6 +117,15 @@ export class ActionComponent extends Component {
                     </div>
                     );
             }
+        } else if (this.props.type === 'postback') {
+            return (
+                <div className='sk-action'>
+                    <button className='btn btn-sk-primary'
+                            onClick={ this.onPostbackClick.bind(this) }>
```

Comment:
oh i'll update to that thing, because i am now handling states, so it needs its own method and scope!